helm/stargate/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for stargate.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository311: "stargateio/coordinator-3_11" 
+  repository40: "stargateio/coordinator-4_0" 
+  repositoryDse628: "stargateio/coordinator-dse-68" 
+  tag: "v2.0.0-BETA-3"
```

Comment:
should we change this to `v2`, I think we do have the `v2` tag as well now..

Reply:
This change is done.